hybrid-cloud-patterns/industrial-edge
# Enable CI for industrial-edge

- Squashed 'common/' changes from 083ae461..30ab3dcb
- Fixup some common/ conflicts
- Add common-test target
- Add some default values.yaml in c/d/external-secrets/values.yaml
- Remove tls cert + key from kafka naked test
- Update datacenter/kafka tests to last changes
- Update datacenter/manuela-tst to latest changes
- Update datacenter/opendatahub to latest changes
- Update datacenter/pipelines to latest changes
- Adapt more tests to make sure they pass
- Add helmlint target
- Add linting/test CI workflow
- Add missing datacenter-external-secrets files
- Also run common-test in CI
- Fix names of workflow to be generic
